README.md

```diff
@@ -8,6 +8,133 @@ It collects data about workflow runs and workflow run jobs from
 * Python 3
 * requests
 
+## API
+
+### fetch.py
+
+SYNOPSIS
+
+    ./multivac/fetch.py [OPTION] [owner/repo]
+
+
+DESCRIPTION
+
+    multivac/fetch.py — download workflow runs GitHub API, workflow run jobs
+    GitHub API and logs. Result stored in `workflow_runs` and `workflow_run_jobs`
+    directories in the root of the project.
+
+
+OPTIONS
+
+    --branch __branch__
+
+            Branch (all if omitted). To collect data about several branches, use
+            this option several times.
+
+    --nologs
+
+            Don't download logs
+
+    --nostop
+
+            Continue till end or rate limit
+
+    --since __N__
+
+            A workflow run list page to start from it. Default: 1
+
+
+EXAMPLE
+
+    Collect all data about workflow runs and workflow run jobs till the end
+    of rate limit from repository `tarantool/multivac` branches `master` and
+    `sample-branch`, but don't collect logs:
+
+```bash 
```

Comment:
If you start lines with $ or #, it's console, not bash

Reply:
Fixed